src/__experimental__/components/search/search.component.js

```diff
@@ -117,6 +118,7 @@ const Search = ({
   return (
     <StyledSearch
       searchWidth={searchWidth}
+      searchButton={searchButton}
```

Comment:
Should this be `showSearchButton`?

Reply:
I agree. It's more descriptive more than `searchButton`. I'll get that changed 👍